client/my-sites/importer/file-importer.jsx

```diff
@@ -79,7 +79,7 @@ class FileImporter extends React.PureComponent {
 		 * This is used for the new Migration logic for the moment.
 		 */
 		if ( isConfigEnabled( 'tools/migrate' ) && overrideDestination ) {
-			return page.redirect( overrideDestination.replace( '%SITE_SLUG%', siteSlug ) );
+			return false;
```

Comment:
It wasn't immediately clear to me why this was needed. Perhaps the comment should explicitly mention that by returning `false` we are allowing the card's href to be followed.